project/api/models.py

```diff
@@ -226,6 +227,7 @@ class SpacetimeVolume(models.Model):
     territory = models.ManyToManyField(AtomicPolygon, related_name="stvs")
     entity = models.ForeignKey(TerritorialEntity, on_delete=models.CASCADE)
     references = ArrayField(models.TextField(max_length=500))
+    visual_center = models.PointField(default=Point(0, 0))
```

Comment:
I would recommend against using a default here unless it was guaranteed to be in the geometry. [ST_PointOnSurface](https://postgis.net/docs/ST_PointOnSurface.html) perhaps